doc/TOPP_tutorial/TOPP_scripting.doxygen

```diff
@@ -34,6 +34,267 @@
 
 //##############################################################################
 
+/**
+        @page TOPP_example_qualitycontrol Qualitycontrol
+
+        @section TOPP_example_qualitycontrol_section Qualitycontrol
+
+        @b Goal: Check the quality of the data.
+
+        The QualityControl TOPP tool is able to check the quality of different aspects if enough data is provided.
+        Depending on the given data this tool executes different metrics that have access to all of their required
+        files (see section 'Metrics'). They will then add new metavalues to the data and write all possiblbe
+        information into the mzTab output file.
+
+        @section TOPP_example_qualitycontrol_section2 Workflow
+
+        @image html TOPP_qualitycontrol.png
+        @image latex TOPP_qualitycontrol.png "" width=10cm
+
+
+        @section  TOPP_example_qualitycontrol_section3 Metrics
+        This Table shows what each of the included metrics does, what they need to be executed and what they add to
+        the data or what they return.
+        @subsection TOPP_example_qualitycontrol_section3_subsection Input Data
+        PostFDR FeatureXML: A FeatureXML after FDR filtering. <br>
+        Contaminants Fasta file: A Fasta file containing the contaminants database. <br>
+        Raw mzML file: An unchanged mzML file. <br>
```

Comment:
the MzCalibration metric requires an mzML which had undergone InternalCalibration. Not mentioned here? All other metrics do not really care which mzML file they get